addons/event/models/event_event.py

```diff
@@ -766,6 +766,32 @@ def _verify_seats_availability(self, slot_tickets):
     # ACTIONS
     # ------------------------------------------------------------
 
+    def action_open_slot_calendar(self):
+        self.ensure_one()
+        now = datetime.now().astimezone(pytz.timezone(self.env.user.tz or 'UTC'))
+        next_hour = now + timedelta(hours=1)
+        return {
+            'type': 'ir.actions.act_window',
+            'name': _('Slots'),
+            'view_mode': 'calendar,list,form',
+            'mobile_view_mode': 'list',
```

Comment:
The focus is very strange at creation on the list view for mobile: but probably not linked to your changes, just noticing. We scroll a lot down and we do not even see the edited row.

<img width="505" height="532" alt="image" src="https://github.com/user-attachments/assets/47727f4f-9ab1-452e-97a7-5da6e038a09f" />